services/datamanager/builtin/builtin.go

```diff
@@ -408,22 +419,15 @@ func (svc *builtIn) initSyncer(ctx context.Context) error {
 	ctx, cancel := context.WithTimeout(ctx, grpcConnectionTimeout)
 	defer cancel()
 	identity, conn, err := svc.cloudConnSvc.AcquireConnection(ctx)
-	if errors.Is(err, cloud.ErrNotCloudManaged) {
-		svc.logger.CDebug(ctx, "Using no-op sync manager when not cloud managed")
-		svc.syncer = datasync.NewNoopManager()
-	}
 	if err != nil {
 		return err
 	}
 
 	client := v1.NewDataSyncServiceClient(conn)
 	svc.filesToSync = make(chan string, svc.maxSyncThreads)
-	syncer, err := svc.syncerConstructor(identity, client, svc.logger, svc.captureDir, svc.maxSyncThreads, svc.filesToSync)
-	if err != nil {
-		return errors.Wrap(err, "failed to initialize new syncer")
-	}
-	svc.syncer = syncer
+	svc.syncer = svc.syncerConstructor(identity, client, svc.logger, svc.captureDir, svc.maxSyncThreads, svc.filesToSync)
```

Comment:
`syncerConstructor` never returned an errors so I made the signature reflect that.